pre-docs/inventory_version2.md

```diff
@@ -120,7 +120,7 @@ The following plugins can be used for targets
 
 * `puppetdb` - Query PuppetDB to populate the targets.
 * `terraform` - Load a Terraform state file to populate the targets.
-* `aws::ec2` - Load running AWS EC2 instances to populate the targets.
+* `aws_inventory` - Load running AWS EC2 instances to populate the targets.
 * `task` - Run a task to discover targets.
 
 #### Config plugins
```

Comment:
We will need to document or remove the examples for the task plugin. 
for example something like 
```
#!/home/cas/.rbenv/shims/ruby
require 'json'
x = {'targets': [{'uri' => 'localhost'}]}
puts x.to_json
```
```
used to work, but now we expect 
```
#!/home/cas/.rbenv/shims/ruby
require 'json'
x = {'value' => ['uri' => 'localhost']}
puts x.to_json
```

Reply:
oops fixed this in the other file but not here.